blocks/init/src/Blocks/custom/accordion/accordion.php

```diff
@@ -9,14 +9,10 @@
 use EightshiftBoilerplateVendor\EightshiftLibs\Helpers\Components;
 
 $manifest = Components::getManifest(__DIR__);
-$manifestBlockName = $manifest['blockName'];
 
 echo Components::render( // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
 	'accordion',
-	array_merge(
-		Components::prop('accordion', $attributes),
-		[
-			'accordionContent' => $innerBlockContent
-		]
-	)
+	Components::prop('accordion', $attributes, [
```

Comment:
This method is either wrongly named here, or in the libs

https://github.com/infinum/eightshift-libs/blob/release/3.1.0/src/Helpers/Components.php#L894

`prop` vs `props`. Just tested it and got a bunch of phpstan errors for missing function.

Reply:
yea this is wrong tnx